### PR TITLE
Do not trigger stale bot on each PR

### PR DESCRIPTION
**Pull request by SiarheiFedartsou**
Just a leftover I forgot to remove after https://github.com/Project-OSRM/osrm-backend/pull/6985

### CODE REVIEW

**Comment by SiarheiFedartsou** [comment]
Merging it as I doubt there will be any concerns...

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 name: 'Close stale issues'
 on:
-  pull_request:
-    branches:
-      - master
+  # uncomment if you want to test changes to this file in PRs CI
```
Comment (DennisOSRM) [inline .github/workflows/stale.yml:3]:
```suggestion
  # NOTE: uncomment if you want to test changes to this file in PRs CI
```

Comment (SiarheiFedartsou) [reply .github/workflows/stale.yml:3]: Well 😄 https://github.com/Project-OSRM/osrm-backend/pull/6986#issuecomment-2218328050

Sorry for this...

Comment (SiarheiFedartsou) [reply .github/workflows/stale.yml:3]: I'll create a separate one.

**Pull request merged**